README.md

```diff
@@ -1,32 +1,283 @@
+Now we will see how to use the `Statement` interface to execute an SQL statement:
 
-## Topic 1
+- **Statement** : An interface that executes SQL statements through a database connection.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+The `Statement` interface has several methods for executing
+different types of SQL statements:
 
-## Topic 2
+- `boolean execute (String SQL)` - Execute a DDL statement such as `CREATE TABLE` and `DROP TABLE`.
+- `int executeUpdate (String SQL)` - Execute a DML statement such as `INSERT`, `UPDATE`, and `DELETE`.
+   The method returns the number of rows affected.
+- `ResultSet executeQuery (String SQL)` - Execute a DQL statement such as `SELECT`. 
+   The method returns a result set, which encapsulates a data table.
+
+This lesson will demonstrate the `execute` and `executeUpdate` methods.
+The `executeQuery` method will be demonstrated in a separate lesson.
+
+## Code along
+
+[Fork and clone](https://github.com/learn-co-curriculum/java-mod-5-jdbc-statement) this lesson to run the sample Java programs.
+
+NOTE: This lesson assumes you created a new PostgreSQL database named `employee_db` as instructed
+in the previous lesson.
+
+## Execute DROP TABLE and CREATE TABLE statements
+
+The steps to execute an SQL CREATE TABLE statement are:
+
+1. Create a `Connection` object.  
+   `Connection connection = DriverManager.getConnection(DB_URL, USER, PASSWORD);`
+2. Use the `Connection` object to create a `Statement` object .  
+   `Statement statement = connection.createStatement();`
+3. Call the `execute` method on the `Statement` object, passing a string containing an SQL DDL statement.    
+   `statement.execute("CREATE TABLE employee (id INTEGER PRIMARY KEY, email TEXT, office TEXT, salary DECIMAL)");`
+
+**NOTE:** Omit the semicolon in the SQL string passed into the `execute` method.
+
+The  `CreateTableStatement` class below demonstrates how to use a `Statement` object
+to execute `DROP TABLE` and `CREATE TABLE` statements. The method
+calls `getConnection`, `createStatement`, and `execute` all may throw an `SQLException` and
+are encapsulated in a `try-catch` block.
+
+```java
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class CreateTableStatement {
+
+   static final String DB_URL = "jdbc:postgresql://localhost:5432/employee_db";
+   static final String USER = "postgres";
+   static final String PASSWORD = "postgres";
+
+   public static void main(String[] args) {
+      try (   //Establish a connection to the database
+              Connection connection = DriverManager.getConnection(DB_URL, USER, PASSWORD);
+              //Create a statement object to execute SQL statements on the database
+              Statement statement = connection.createStatement();
```

Comment:
Suggestion: In the File IO lesson where I explained the try-with-resources statement, I didn't really mention you could have multiple resources since I was only opening one resource. You might want to briefly explain here you can open multiple resources and that they will be closed in the reverse order they were defined if this is the first time students will be seeing a try-with-resources with multiple resources.

Reply:
Ok, lesson updated with an explanation of the closing order.